android/build.gradle

```diff
@@ -111,7 +109,6 @@ afterEvaluate { project ->
     task androidJavadoc(type: Javadoc) {
```

Comment:
I reckon we could delete this whole task block.. can we try it out @lmcjt37 ?

Reply:
@fcglo is picking this up now I believe 😉